packages/@react-types/dialog/src/index.d.ts

```diff
@@ -37,10 +37,12 @@ export interface SpectrumDialogProps extends DOMProps, StyleProps {
    */
   slots?: Slots,
   /** The size of the Dialog. Only applies to "modal" type Dialogs. */
```

Comment:
Had to bring back the `Slots` type since Dialog is no longer split into ModalDialog and Dialog, meaning that it will appear in the docs again. Not sure of a good way to exclude it as requested without splitting the component into two again